sentences/en/scene_HassTurnOn.yaml

```diff
@@ -4,8 +4,17 @@ intents:
     data:
       - sentences:
           - "[activate|<turn>] <name> [scene] [on]"
+          - "(change|transition) to <name> [scene]"
         requires_context:
           domain: scene
         slots:
           domain: scene
         response: scene
+      - sentences:
+          - "[activate|<turn>] <area> <name> [scene] [on]"
+          - "[activate|<turn>] <name> [scene] (in|on|at) <area>"
+          - "(change|transition) [to] <area> [to] <name> [scene]"
```

Comment:
Why the double `[to]`? Could you please provide an example sentence?

Reply:
Yeah, it was structured this way to avoid two nearly identical senteces, I'm happy to break it out into seperate ones or revise it if you feel there is a way to make it clearer though. 

Examples: 
- "change **_to_** the kitchen party mode scene"
- "change the kitchen **_to_** party mode"

Reply:
```suggestion
          - "(change|transition) ([to] <area> <name>|<area> to <name>) [scene]"
```